### PR TITLE
add org.gtk.Gtk3theme.Mojave-light

### PR DESCRIPTION
**Pull request by elsiehupp**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub.
> See: https://github.com/vinceliuice/Mojave-gtk-theme/issues/142#issuecomment-805106552

- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
> This is the weird one. I obviously don't own the domain gtk.org, but the way Flatpak does themes, GTK themes have to start with `org.gtk.Gtk3theme`, and the text immediately after has to match the name of a non-sandboxed theme that has already been installed and activated by the user. I tried using the ID `io.github.elsiehupp.Mojave-dark`, and `appstream-util validate` checked out, but `flatpak-builder` gave me the following error:
> ```
> error: No extension point matching io.github.elsiehupp.Mojave-light in runtime/org.freedesktop.Sdk/x86_64/18.08
> ```
> So, yeah... if this flatpak is ineligible for Flathub, I suppose I could always modify the upstream installer script to obtain the necessary results. For what it's worth, [the Snap Store seems to have been perfectly happy to accept a Snap version of the full set of themes](https://snapcraft.io/mojave-themes), the only problem being that Flatpak apps don't see them.

- [ ] Any additional patches or files have been submitted to the upstream projects concerned.
> The only changes I made were figuring out how to package the theme as a flatpak. Because of the way Flatpak seems to handle themes (in that the name of the theme is part of the ID), and `Mojave-gtk-theme` comprises eight different sub-themes, there isn't really much point in submitting the changes to the upstream since I would be submitting at least eight different flatpaks. If, however, I end up making changes to the flatpak so that it uses the upstream repository directly, that might involve making contributions to the upstream (if only an install script).

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by nedrichards** [comment]
Using the org.gtk namespace for themes is fine.

**Comment by nedrichards** [comment]
bot, build org.gtk.Gtk3theme.Mojave-light

**Comment by barthalion** [comment]
You need to add a file like this one: https://github.com/flathub/org.gtk.Gtk3theme.Breeze/blob/master/flathub.json

File: org.gtk.Gtk3theme.Mojave-light.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (barthalion) [inline org.gtk.Gtk3theme.Mojave-light.appdata.xml:1]: Note we don't display themes on https://flathub.org so usually appdata is more barebone like https://github.com/flathub/org.gtk.Gtk3theme.Pop/blob/master/org.gtk.Gtk3theme.Pop.appdata.xml

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,15 +0,0 @@
-### Submitting a new package
```
Comment (barthalion) [inline CONTRIBUTING.md:1]: Ideally base your branch on top of the `new-pr` branch for the future PRs.

**Pull request closed without merging**